Update list.json

mizterp opened the pull request:
Reorganized & removed duplicates

FGRibreau commented:
Hello @mizterp,

Thanks for your contribution, what duplicates did you exclude? 
Can someone check that everything's fine with this PR?

mizterp commented:
I realized it would be quite time consuming to mark the duplicates in each list, so I just created one single list, added more (which I found on github), removed duplicates in one go and sorted it.

But if the goal is to maintain multiple lists (even with all the duplicates) then it might be best to ignore this PR.
